portal_odoo_debranding/views/portal_templates.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="frontend_layout_debrand" inherit_id="portal.frontend_layout">
```

Comment:
Put priority="9999" for avoiding problems with the replaces

Reply:
@pedrobaeza I applied what @simahawk suggested at https://github.com/OCA/website/pull/729#discussion_r449424554 as more inheritance compliant.
What do you think?